.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,100 @@
+name: Bug report
+description: Create a report to help us improve CodeIgniter
+title: "Bug: "
+labels: ['bug']
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Before you begin, please
+        ensure that there are no existing issues, whether still open or closed, related
+        to your report. If there is, your report will be closed promptly.
+
+  - type: dropdowm
+    id: php-version
+    attributes:
+      label: PHP Version
+      description: Which PHP versions did you run your code?
+      multiple: true
+      options:
+        - '7.3'
+        - '7.4'
+        - '8.0'
+        - '8.1'
+    validations:
+      required: true
+
+  - type: input
+    id: codeigniter-version
+    attributes:
+      label: CodeIgniter4 Version
+    validations:
+      required: true
+
+  - type: dropdown
+    id: operating-systems
+    attributes:
+      label: Which operating systems have you tested for this bug?
+      description: You may select more than one.
+      multiple: true
+      options:
+        - macOS
+        - Windows
+        - Linux
+    validations:
+      required: true
+
+  - type: dropdown
+    id: server
+    attributes:
+      label: Which server did you use?
+      options:
+        - apache
+        - cli
+        - cli-server (PHP built-in webserver)
+        - cgi-fcgi
+        - fpm-fcgi
+        - phpdbg
+    validations:
+      required: true
+
+  - type: input
+    id: database
+    attributes:
+      label: Database
+    validations:
+      required: false
+
+  - type: textarea
+    id: description
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
```

Comment:
This should ask only what happened. 

Not to be expected to happen.
It is asked in `Expected Output`.